extensions/transform/org.eclipse.smarthome.transform.xslt/README.md

```diff
@@ -28,10 +44,60 @@ transform/helloworld.xsl:
 </xsl:stylesheet>
 ```
 
-output:
+**Output XML**
 
 ```xml
 <reRoot><reNode>hello world</reNode></reRoot>
 ```
 
-Other examples may be found [here](https://en.wikipedia.org/wiki/XSLT#XSLT_examples).
\ No newline at end of file
+## Advanced Example
+
+**input XML**
+
+This example has an namespace defined, as you would find in real world applications, which has to be matched in the rule.
+
+* The tag `<PTZStatus>` contains a namespace `xmlns=` 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<PTZStatus version="2.0" xmlns="http://www.hikvision.com/ver20/XMLSchema">
+	<AbsoluteHigh>
+		<elevation>0</elevation>
+		<azimuth date="Fri, 18 Dec 2009 9:38 am PST" >450</azimuth>
+		<absoluteZoom>10</absoluteZoom>
+	</AbsoluteHigh>
+</PTZStatus>
+```
+
+**transform/azimut.xsl**
+
+In the rule the tag `<xsl:stylesheet>` has to have the  `xmlns:xsl="http://www.w3.org/1999/XSL/Transform"` attribute and a second attribut `xmlns:` which has to be the same as the namespace for the input document. In the rule each step traversed along the path to the next tag has to prepend the `xmlns` namespace, here defined as `h`.
```

Comment:
Please break the text down to one line per Sentence.
We do it this way for better parsing and keeping git changes well documented.